### DISCUSSION TITLE
Configurable screen size

### DISCUSSION THREAD

**Discussion by tpetry** [Xammie/mailbook#43] [Ideas]
Mailbook can show a mail for different screen sizes. However the default for a smartphone (448px) is much too high. The value depends on the minimum device you set as expected:

* iPhone SE: 375px
* iPhone XR: 414px
* iPhone 12 Pro: 390px
* Pixel 5: 393px
* Saamsung Galaxy S8+: 360px
* Samsung Galaxy S20 Ultra: 412px

**Comment by Xammie** [comment]
Are you suggesting that the mobile size should be reduced to one of those examples?

**Comment by tpetry** [comment]
I thought of a setting in the config that is defaulting to one of those values. Maybe the iPhone SE size?
